otter/cloud_client.py

```diff
@@ -269,13 +269,12 @@ def _default_throttler(clock, stype, method):
     # no more than that are executed by a node, plus serialization of requests
     # will make it quite a bit lower than that.
 
-    cloud_client_config = config_value('cloud_client')
-    if cloud_client_config is None:
-        cloud_client_config = {}
-    throttling_config = cloud_client_config.get('throttling', {})
+    throttling_config = config_value('cloud_client.throttling')
+    if throttling_config is None:
+        return None
+
     create_server_delay = throttling_config.get('create_server_delay', 1)
```

Comment:
I would just get rid of these defaults here and rely on the config now. with your change I think the behavior is a bit confusing, in that if you put a `throttling: {}` in the config it will throttle with 1 and 0.4, I think it's better now to just not have any defaults and rely on the config to specify it.